common/src/gas_provider/node.rs

```diff
@@ -19,13 +19,31 @@
 use super::*;
 use codec::MaxEncodedLen;
 
-// TODO remove children refs for Unspecified (https://github.com/gear-tech/gear/issues/1179)
 #[derive(Clone, Decode, Debug, Encode, MaxEncodedLen, TypeInfo, PartialEq, Eq)]
 pub enum GasNodeType<ExternalId, Id, Balance> {
```

Comment:
We have decided, that `consumed` is specific field to `External` and `SpecifiedLocal` only
But the general node can also have `.consumed()` method

Reply:
done